### DISCUSSION TITLE
Q: Party Pack Application Behavior After Ending a Game

### DISCUSSION THREAD

**Discussion by MisterAlexEsquire** [AlexisL61/JackboxPatcherFR#411] [Q&A] [answered]
Mild disappointment while trying out the official Megapicker is what brought me to find Jackbox Utility, and as it is without any changes, I already like JB Utility more. That said there is one thing that I noticed about Megapicker that seemed pretty convenient.

**When exiting an individual game (i.e. selecting Back and then confirming), should the active Party Pack return to its home screen or close entirely and return the user to Jackbox Utility?**

As I tested in Megapicker, a selected game would launch directly (as it does with JB Utility), but then when exiting that game, the whole Party Pack closed and returned to Megapicker. With JB Utility, the Party pack did not close, and I was instead brought to the Party Pack's game selection menu. 

I don't actually know if there's a parameter or anything that would even allow for JB Utility and other third-party launchers to replicate the same behavior, but I figured I'd ask. Also, I would note that I'm using release version 1.3.7+2 since the latest version crashes at startup for me.

**Comment by AkiraArtuhaxis** [comment, marked as answer]
This already happens in 1.4.0. This version introduces the same kind of launching that the Megapicker uses. Unfortunately, this feature is only reserved to Steam users, as only these versions were updated to support this (that's the main reason why all the Party Packs were updated on Steam). As long as we figure out why 1.4.0 crashes for so many people, if you use Steam you already got that covered.

**Reply by MisterAlexEsquire** [reply]
Ah, gotcha. Thanks for clarifying!

_Edit: Okay, now that I've got 1.4.0 running, I'm seeing the auto-close behavior. It looks like Party Pack 10 is the only one that doesn't behave that way (the pack launches normally to the main game select menu), and that may change if I get an update for that game._
_Note: I have the Party Packs installed via Steam, and I'm running Jackbox Utilty directly from a pinned Start Menu shortcut._

**Reply by AkiraArtuhaxis** [reply]
Thanks for the heads up. Pack 10 games are now fixed and should launch like the rest of the games.

**Comment by AkiraArtuhaxis** [comment]
That being said, before this was a thing I made the [Jackbox Mouse](https://github.com/orgs/JackboxUtility/discussions/257), which lets me quit out of the Party Pack quickly to return to the Utility. Only problem is there's no confirmation to close it, so there's bound to be a misinput here and there.

**Reply by MisterAlexEsquire** [reply]
Oh, that's pretty handy. Thank you!
